lecture/forms.py

```
@@ -0,0 +1,20 @@
+from django import forms
+from django.utils.text import slugify
+from .models import Lecture  # Import your model
+
+class LectureForm(forms.ModelForm):
+    class Meta:
+        model = Lecture
+        fields = ['title', 'pdf_file']
+
+    def save(self, commit=True):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

(ANN201)